docs/development/releasing.rst

```diff
@@ -225,18 +225,25 @@ packages that use the full bugfix/maintenance branch approach.)
       but this should *not* be done, as it might push up some unintended tags.
 
 #. If this is a release of the current release (i.e., not an LTS supported along
-   side a more recent version), update the "stable" branch to point to the new
+   side a more recent version), update the "stable" tag to point to the new
    release::
 
-      $ git checkout stable
-      $ git reset --hard v<version>
+      $ git checkout v<version>
+      $ git tag -s --force stable
       $ git push upstream stable --force
 
-#. Update Readthedocs so that it builds docs for the version you just released.
-   You'll find this in the "admin" tab, with checkboxes next to each github tag.
-   Also verify that the ``stable`` Readthedocs version builds correctly for
-   the new version (it should trigger automatically once you've done the
-   previous step).
+#. If this release is an LTS release, update the "LTS" tag to point to the
+   new release::
+
+      $ git checkout v<version>
+      $ git tag -s --force LTS
+      $ git push upstream LTS --force
```

Comment:
I'm a bit scare by the --force - I don't think we should ever force push. It would be safer to delete the tag and push a new one without forcing.

Reply:
Sure, that should also work, though we had the force pushing for stable before, too.

Reply:
Yep I just saw that - though now they are tags we don't need that anymore

Reply:
Could you have another look? I'm not convinced the new way is that much safer, deleting remote stuff always freaks me out.